style.css

```diff
@@ -56,10 +57,195 @@ nav {
   display: flex;
 }
 
-.links a {
+.links li {
   margin-right: 20px;
 }
 
-.liks a img {
+.links a img {
   width: 100%;
 }
+
+/***************** The project sections styles ***************/
+.projects {
+  background-color: white;
+  border-radius: 10px;
+  width: 90%;
+  margin: 5px auto;
+  padding: 1rem;
+  margin-bottom: 4rem;
+}
+
+.projects h2 {
+  text-transform: capitalize;
+  margin-bottom: 0.7rem;
+  font-size: 32px;
+  line-height: 44px;
+  font-weight: 700;
+}
+
+.projects ul {
+  display: flex;
+  margin-bottom: 0.7rem;
+}
+
+.projects li {
+  margin-right: 1rem;
+  font-size: 13px;
+  color: #7a869a;
+  list-style-position: inside;
+  text-transform: capitalize;
+  font-weight: 600;
+}
+
+.projects p {
+  margin-bottom: 0.7rem;
+}
+
+#dark {
+  list-style: none;
+  color: black;
+}
+
+.stack {
+  display: flex;
+  margin-bottom: 0.7rem;
+}
+
+.stack li {
+  color: #6070ff;
+  font-size: 12px;
+  background-color: #ebebff;
+  list-style: none;
+  padding: 5px 8px;
+  border-radius: 5px;
+}
+
+button {
+  margin: 0.7rem 0;
+  color: #396df2;
+  outline: none;
+  background: transparent;
+  font-size: 17px;
+  padding: 5px;
+  border: 1px solid #396df2;
+  border-radius: 5px;
+  font-family: 'Poppins', sans-serif;
+}
```

Comment:
Your buttons look great. Kindly as well implement the button interactions as given by the figma design.

Figma button interactions:
![image](https://user-images.githubusercontent.com/20554583/205286002-2de080c5-9618-4346-8678-e51e511ec05f.png)